protocols/ecmd/via_usart/ecmd_usart.c

```diff
@@ -52,7 +64,9 @@ ecmd_serial_usart_init(void) {
 
   RS485_TE_SETUP;             // configure RS485 transmit enable as output
   RS485_DISABLE_TX;           // disable RS485 transmitter
+#ifdef ECMD_USART_NEED_INIT
```

Comment:
The RS485 pin initialisation has already be done in debug_init_uart. Just move the if -statement two lines up.